lib/utils.js

```diff
@@ -134,8 +134,9 @@ function parseStack(err, cb) {
             lineno: line.getLineNumber(),
             'function': getFunction(line),
         }, isInternal = line.isNative() ||
-                        (frame.filename[0] !== '/' &&
-                         frame.filename[0] !== '.');
+                        (frame.filename[0] !== path.sep &&
+                         frame.filename[0] !== '.' &&
+                         frame.filename.indexOf(':\\') != 1);
```

Comment:
Is this supposed to be `:\\` or `:\`? Also, can you switch it to be a strict inequality check? `!==`